src/Microsoft.VisualStudio.Validation/Requires.cs

```diff
@@ -244,7 +246,7 @@ public static void NotNullEmptyOrNullElements<T>([ValidatedNotNull, NotNull]IEnu
         /// <param name="parameterName">The name of the parameter to include in any thrown exception.</param>
         /// <exception cref="ArgumentException">Thrown if the tested condition is false.</exception>
         [DebuggerStepThrough]
-        public static void NullOrNotNullElements<T>(IEnumerable<T> values, string parameterName)
+        public static void NullOrNotNullElements<T>(IEnumerable<T>? values, string? parameterName)
```

Comment:
```suggestion
        public static void NullOrNotNullElements<T>(IEnumerable<T?>? values, string? parameterName)
```

Reply:
Same as above: I can't do that because `T` is allowed to be a struct. But I'm going the other way anyway.